server/src/main/java/org/opensearch/common/settings/ClusterSettings.java

```diff
@@ -457,6 +457,7 @@ public void apply(Settings value, Settings current, Settings previous) {
                 NetworkService.TCP_CONNECT_TIMEOUT,
                 IndexSettings.QUERY_STRING_ANALYZE_WILDCARD,
                 IndexSettings.QUERY_STRING_ALLOW_LEADING_WILDCARD,
+                IndexSettings.TIME_INDEX_MERGE_POLICY,
```

Comment:
We refer to data streams as time series (in DataStream fe), would be good to stay consistent:

```suggestion
                IndexSettings.TIMESERIES_INDEX_DEFAULT_MERGE_POLICY,
```

Reply:
I intentionally refrained from using `timeseries` as log data is richer than simple timeseries data. However, I do not have very strong opinion on this naming convention and as you said that we have been using `timeseries` at several places in the codebase, i will stick to it as well.